laika/examples/config.json

```diff
@@ -1,11 +1,11 @@
 {
   "timezone": "America/Sao_Paulo",
   "include": [
-    "examples/drive_config.json",
```

Comment:
👍 i was reviewing locally to understand how this was being included, but given that we are moving the configuration inside laika/examples this is solved.